### PR TITLE
Adds a rudimentary navigation menu tab as well as four scenes. (#3)

### LINKED ISSUE #3: Set up app navigation
Ability to navigate between pages of the app

### PR DESCRIPTION
**Pull request by goccert25**
Resolves #3 

The menu tab component was found in an online library. It fits to the platform in that for android, the menu tab will be on top, while for iOS it will be at the bottom. The menu tab can be found and modified inside the MenuTab.js file. The scenes are essentially the different pages of the app. They are found inside the scenes folder.

Once this is merged with the develop branch, the app in develop branch will have 4 pages. It will also have the ability to switch between the pages by using the navigation bar at the bottom (iOS) or at the top (android). 

The pages are extremely rudimentary, as is the navigation bar. We still need to 
- Beautify the menu tab
- Implement what exactly we want in each page

### CODE REVIEW

File: MenuTab.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import ScheduleScene from './scenes/ScheduleScene';
+import CompanyInfoScene from './scenes/CompanyInfoScene'
+import MapScene from './scenes/MapScene'
+import MentorsScene from './scenes/MentorsScene'
+
+//credit to/documentation at
+//https://github.com/skv-headless/react-native-scrollable-tab-view
+
+var ScrollableTabView = require('react-native-scrollable-tab-view');
```
Comment (jdai8) [inline MenuTab.js]: I'd prefer ES6 `import` over `require`.

File: MenuTab.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+//credit to/documentation at
+//https://github.com/skv-headless/react-native-scrollable-tab-view
+
+import ScrollableTabView from 'react-native-scrollable-tab-view';
+
+export default class MenuTab extends Component {
+
+  render() {
+    var menuTabPosition;
```
Comment (adanoff) [inline MenuTab.js]: use `let` instead of `var`.

File: scenes/CompanyInfoScene.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+export default class CompanyInfoScene extends Component {
+  static get defaultProps() {
```
Comment (adanoff) [inline scenes/CompanyInfoScene.js:5]: I'd prefer this to be outside of the class, in the form of

``` javascript
CompanyInfoScene.defaultProps = {
...
}
```

but that's a matter of personal preference. You should be specifying prop types though.

**Comment by adanoff** [comment]
Other than my comments, looks good enough on iOS in my opinion. Future changes I see us needing:
- there should be one `Scene` component that we use to set up e.g. the layout of each scene (headings, etc) so we can ensure consistency there

Also, in general I think we should stay away from shit we find on npm as much as possible unless there's a very good reason not to, but in this case this library seems solid.

**Comment by adanoff** [comment]
One other concern about the 3rd party thing, we need to be sure we can style it. If not there _is_ built in navigation capabilities with react native, we just need to make the components that use it. It probably wouldn't be that hard, if we'd be willing to sacrifice the swipe-to-transition.

File: MenuTab.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { View, Text, Platform } from 'react-native';
```
Comment (jdai8) [inline MenuTab.js]: We can take out `View` and `Text` since they're never used

File: MenuTab.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+import ScheduleScene from './scenes/ScheduleScene';
+import CompanyInfoScene from './scenes/CompanyInfoScene'
```
Comment (jdai8) [inline MenuTab.js]: semicolons

**Pull request merged**